Thorium-Sim/thorium-nova
# Officers log

## Description

Adds the officers log card.

## Related Issue








Closes #148 

## How do you know the changes work correctly?






Manual Testing
- Start a flight
- Click "Cards" in the top menubar
- Choose the officers log card
- Click "New Log"
- Type out a log
- Click "Save". The log should appear in the list on the left.

## Screenshots (if appropriate):
<img width="1092" alt="Screen Shot 2021-12-10 at 8 22 33 AM" src="https://user-images.githubusercontent.com/6558157/145685226-a9f7851c-4cef-411c-ad16-21d616999b1d.png">

## Types of changes



      change)

## Checklist:
